### PR TITLE
Play start and guides workflows

### PR DESCRIPTION
**Pull request by shkirando**
More useful option in the `:play start' frame, links between guides and help frames

### CODE REVIEW

**Comment by shkirando** [comment]
Screenshot 2020-10-01 at 11 59 17
Screenshot 2020-10-01 at 11 58 40
Screenshot 2020-10-01 at 11 58 27

File: src/browser/documentation/guides/movie-graph.jsx [hunk 1]
```diff
@@ -43,6 +47,30 @@ const slides = [
...
+      </p>
+      <div>
+        If you don't want to modify your data:
+        <ol>
+          <li>
+            Switch to system database{' '}
+            <TextCommand command={'use system'}></TextCommand>
+          </li>
+          <li>
+            Create a new databse{' '}
```
Comment (OskarDamkjaer) [inline src/browser/documentation/guides/movie-graph.jsx:63]: Small spelling error

File: src/browser/documentation/guides/movie-graph.jsx [hunk 2]
```diff
@@ -43,6 +47,30 @@ const slides = [
...
+            <pre
+              mode="cypher"
+              className="pre-scrollable code runnable"
+              style={{ width: 'fit-content' }}
+            >
```
Comment (OskarDamkjaer) [inline src/browser/documentation/guides/movie-graph.jsx:68]: @shkirando  It'd be nice if the commands were runnable easily. So I'm thinking we could have it be: `create database movies` and then `:use movies`. As steps 2 and 3

File: src/browser/modules/Frame/FrameAside.jsx [hunk 3]
```diff
@@ -21,14 +21,15 @@
...
+import { DARK_THEME, LIGHT_THEME } from 'shared/modules/settings/settingsDuck'
 
 class FrameAside extends Component {
   render() {
     const { subtitle } = this.props
     let { title } = this.props
 
     // Use logo as title if title is only Neo4j
-    if (title === 'Neo4j') {
+    if ((title === 'Neo4j', LIGHT_THEME)) {
```
Comment (OskarDamkjaer) [inline src/browser/modules/Frame/FrameAside.jsx:32]: Using , here is a bit confusing as to what is meant. I'd say use && or || instead

File: src/browser/modules/Frame/FrameAside.jsx [hunk 4]
```diff
@@ -37,6 +38,13 @@ class FrameAside extends Component {
         />
       )
     }
+    if ((title === 'Neo4j', DARK_THEME)) {
+      ;<img
```
Comment (OskarDamkjaer) [inline src/browser/modules/Frame/FrameAside.jsx:42]: This lone seems to be missing a title =

I can't get it to work at least :)

**Pull request merged**